spec/21-http_response_header_format.md

```diff
@@ -1,113 +1,104 @@
 ```
 
-The dash (`-`) character is used as a delimiter between fields.
+### Metric Name
 
-#### version
+Metric name: `trace`
 
-``` abnf
-version         = 2HEXDIGLC   ; this document assumes version 00. Version ff is forbidden
-```
+The metric {{PerformanceServerTiming/name}} is [ASCII case-insensitive](https://infra.spec.whatwg.org/#ascii-case-insensitive). That is, `trace`, `Trace`, and `TRACE` are considered the same metric.
 
-Version (`version`) is an 8-bit unsigned integer value, serialized as an ASCII string with two characters. Version 255 (`"ff"`) is invalid. This document specifies version 0 (`"00"`) of the `traceresponse` header.
+In order to increase interoperability across multiple protocols and encourage successful integration, tracing systems SHOULD encode the metric name as [ASCII lowercase](https://infra.spec.whatwg.org/#ascii-lowercase).
 
-#### version-format
+### Metric Description Format
 
-The following `version-format` definition is used for version `00`.
+This section uses the Augmented Backus-Naur Form (ABNF) notation of [[!RFC5234]], including the DIGIT rule from that document. The `DIGIT` rule defines a single number character `0`-`9`.
 
-``` abnf
+```abnf
+HEXDIGLC         = DIGIT / "a" / "b" / "c" / "d" / "e" / "f" ; lowercase hex character
+value            = version "-" version-format
+version          = 2HEXDIGLC   ; this document assumes version 00. Version ff is forbidden
 version-format   = trace-id "-" child-id "-" trace-flags
 trace-id         = 32HEXDIGLC  ; 16 bytes array identifier. All zeroes forbidden
 child-id         = 16HEXDIGLC  ; 8 bytes array identifier. All zeroes forbidden
-trace-flags      = 2HEXDIGLC   ; 8 bit flags. See below for details
+trace-flags      = 2HEXDIGLC   ; 8 bit flags.
 ```
 
 #### trace-id
 
-The format and requirements for this are the same as those of the trace-id field in the `traceparent` request header.
+The format and requirements for this are the same as those of the `trace-id` field in the `traceparent` request header. This is a required parameter.
 
-For details, see the trace-id section under [traceparent Header Field Values](#traceparent-header-field-values).
+For details, see the `trace-id` section under [traceparent Header Field Values](#traceparent-header-field-values).
 
 #### child-id
 
-This is the ID of the operation of the callee (in some tracing systems, this is known as the `span-id`, where a `span` is the execution of a client request) and is used to uniquely identify an operation within a trace. It is represented as an 8-byte array, for example, `00f067aa0ba902b7`. All bytes as zero (`0000000000000000`) is considered an invalid value.
+This is the span ID of the server operation. It is represented as an 8-byte array, for example, `00f067aa0ba902b7`. An all-zero child ID (`0000000000000000`) is an invalid value. Tracing systems MUST ignore the trace context metric when the child id is invalid (for example, if it contains non-lowercase hex characters).
```

Comment:
I don't normally like to bikeshed names too much, but to me this descriptive text suggests that the name of this field should be changed to `span-id`.  It would read much more cleanly and sidestep any questions of semantics ("child of.... what?")